examples/gnrc_networking-subnets/setup_taps.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+PREFIX=2001:db8::/60
+
+RIOTBASE=../..
+RIOTTOOLS=$RIOTBASE/dist/tools
+TOPOLOGY=tapology.txt
+
+setup() {
+    # add address to loopback so we can test reaching an
+    # address outside the test net
+    sudo ip a a fd00:dead:beef::1/128 dev lo
+
+    echo "creating tap interfaces"
+    i=0
+    sed '/^#/d' "$1" | while read -r level num; do
+        # shellcheck disable=SC2091 # we actually want to execute the output here.
```

Comment:
Mhhh apparently the comment makes a problem. Can you add it above the shellcheck directive?